test/mocha.opts

```diff
@@ -5,4 +5,5 @@
 --require react-native-mock/mock.js
 --require react-native-svg-mock/mock
 --require babel-polyfill
+--require jsdom-global/register
```

Comment:
This was needed to make sure that the `window` object is available to our tests. It's not actually used anywhere but just loads a mocked DOM for the tests. `redux-persist` will use `window.localStorage` when running the tests.